example/src/Root.js

```diff
@@ -10,12 +10,15 @@ import CardFormScreen from './scenes/CardFormScreen'
 import CustomCardScreen from './scenes/CustomCardScreen'
 import CustomBankScreen from './scenes/CustomBankScreen'
 import CardTextFieldScreen from './scenes/CardTextFieldScreen'
+import SourceScreen from './scenes/SourceScreen'
 import testID from './utils/testID'
 
 stripe.init({
   publishableKey: '<PUBLISHABLE_KEY>',
   merchantId: '<MERCHANT_ID>',
   androidPayMode: 'test',
+  country: 'US',
```

Comment:
We don't use `country`/`currency` inside `init` method in native part, so please remove it here:
```
– country: 'US',
– currency: 'usd',
```